src/Accounts/Accounts/AzureRmAlias/Mappings.json

```diff
@@ -616,7 +616,11 @@
     "Remove-AzVmssVMDataDisk": "Remove-AzureRmVmssVMDataDisk",
     "Get-AzVmssDiskEncryptionStatus": "Get-AzureRmVmssDiskEncryptionStatus",
     "Get-AzVmssVMDiskEncryptionStatus": "Get-AzureRmVmssVMDiskEncryptionStatus",
-    "Repair-AzVmssServiceFabricUD": "Repair-AzureRmVmssServiceFabricUD"
+    "Repair-AzVmssServiceFabricUD": "Repair-AzureRmVmssServiceFabricUD",
+    "New-AzProximityPlacementGroup": "New-AzureRmProximityPlacementGroup",
```

Comment:
We should not add AzureRM mappings for new cmdkets - these cmdlets will never exist in AzureRm